jenkins/job-configs/bootstrap-maintenance.yaml

```diff
@@ -92,10 +96,14 @@
         frequency: '@daily'
         job-name: maintenance-daily
         repo-name: 'k8s.io/test-infra'
-        timeout: 30
+        maintenance-all-timeout: 1800
+        maintenance-timeout: 30
+        bootstrap-timeout: 30
     - hourly:
         branch: master
         frequency: '@hourly'
         job-name: maintenance-hourly
         repo-name: 'k8s.io/test-infra'
-        timeout: 30
+        maintenance-all-timeout: 180
+        maintenance-timeout: 30
```

Comment:
Maintenance-timeout should be 20m more than bootstrap-timeout, to allow bootstrap time to clean up after itself: https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L126